self-tests.yml

```diff
@@ -663,6 +663,24 @@ tasks:
     commands:
       - func: get-project-and-modules
       - func: verify-merge-function-update
+  - name: verify-gqlgen
+    tags: ["linter"]
+    commands:
+      - func: get-project-and-modules
+      - command: subprocess.exec
```

Comment:
Instead of using make directly, you should use the function `run-make` we have. It correctly sets env variables (which is why the task is failing on this PR).